src/tests/test_rest_framework_dso/test_serializers.py

```diff
@@ -72,3 +87,18 @@ def test_pagination_many(api_rf, movie):
             "category": [{"name": "bar"}],
         },
     }
+
+
+@pytest.mark.django_db
+def test_location(location):
+    """Prove that the serializer transforms crs"""
+
+    request = DummyRequest(CRS.from_string("EPSG:4258").srid)
+    serializer = LocationSerializer(
+        instance=location, fields_to_expand=[], context={"request": request,},
+    )
+    data = serializer.data
+    rounder = lambda p: [round(c, 6) for c in p]
+    assert rounder(data["geometry"]["coordinates"]) == rounder(
+        [3.313687692711974, 47.97485812241689]
+    )
```

Comment:
In `tests.test_rest_framework_dso/test_views.py` kan je prima ook de volledige request/response unit testen. Nu is nog onzeker of die parsing ook werkt.